crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -260,7 +260,7 @@ impl AcirContext {
         typ: AcirType,
     ) -> Result<AcirVar, RuntimeError> {
         let inputs = vec![AcirValue::Var(lhs, typ.clone()), AcirValue::Var(rhs, typ)];
-        let outputs = self.black_box_function(BlackBoxFunc::XOR, inputs)?;
+        let outputs = self.black_box_function(BlackBoxFunc::XOR, inputs, 1)?;
```

Comment:
the 1 is a little magic

Reply:
Yeah it is a bit magic. But this PR ultimately removes the multiple magic numbers used inside of `black_box_function` for parsing the singular inputs array. Having to specify the output count for a simple operation like XOR I think is ok